modules/javafx.graphics/src/main/java/com/sun/glass/ui/monocle/MonocleTimer.java

```diff
@@ -34,6 +34,9 @@
  * Monocle implementation class for Timer.
  */
 final class MonocleTimer extends Timer {
+    private static final String THREAD_NAME = "Monocle Timer";
+    private static final int POOL_SIZE = 1;
```

Comment:
I don't think there is a usecase where POOL_SIZE would be different from 1, so this could be skipped?

Reply:
Thanks, Johan. I'll remove it.

By the way, I added the thread name constant instead of calling `getClass().getSimpleName()` just so the name would appear similar to the other threads in the system, shown below.

![javafx-threads-2020-06-27](https://user-images.githubusercontent.com/1413266/85926564-1eff4c00-b855-11ea-98f3-1a2ec86d1348.png)